src/app/pages/register/register.component.html

```diff
@@ -12,431 +12,467 @@
     </app-alert>
   }
 
-  <!-- Step 1: General information -->
-  @if (currentStep() === 1) {
+  <!-- Registration Page -->
+  @if (!isRegistrationComplete()) {
     <div
-      class="flex w-full flex-1 flex-col items-center justify-start bg-gray-50 p-12"
+      class="flex w-full flex-1 flex-col items-center justify-start bg-gray-50"
     >
-      <div class="flex w-full max-w-7xl flex-1 flex-col gap-6">
-        <h1 class="text-left text-3xl font-semibold text-gray-900">
-          My BioCommons Access
-        </h1>
-        <p class="text-left font-light text-gray-600">
-          My BioCommons Access The BioCommons Access account provides
-          researchers with a single, secure login to use national life science
-          platforms such as Galaxy Australia and the Bioplatforms Data Portal.
-          <br /><br />
-          With this account, you can instantly access open data, tools, and
-          workflows that support bioinformatics research across Australia. It
-          simplifies access by connecting multiple services under one trusted
-          identity, making it easier to move between platforms and collaborate
-          with others in the research community. <br /><br />
-          The BioCommons Access account is designed to give researchers seamless
-          entry to shared digital resources, helping you focus on your science
-          rather than managing multiple logins or registrations.
-        </p>
-      </div>
-    </div>
-  }
+      <!-- Progress Bar - Horizontal and Sticky -->
+      <div class="sticky top-0 z-30 w-full bg-white shadow">
+        <div class="mx-auto max-w-4xl px-6 pb-6 pt-8">
+          <div class="flex items-start justify-between">
+            @for (section of sections; track section.id; let idx = $index) {
+              <div class="relative flex flex-1 flex-col items-center">
+                <div class="flex w-full items-center">
+                  <!-- Circle/Check indicator -->
+                  <div class="flex w-full justify-center">
+                    <a
+                      [href]="'#' + section.id"
+                      class="group cursor-pointer"
+                      (click)="scrollToSection($event, section.id)"
+                    >
+                      <div
+                        class="relative z-10 flex h-8 w-8 flex-shrink-0 items-center justify-center rounded-full border-2 bg-white transition-all"
+                        [ngClass]="{
```

Comment:
Can this show a more obvious colour/icon if the section is invalid/incomplete? It wasn't obvious to me that the dash indicated an incomplete section